views/partials/header.ejs

```diff
@@ -35,6 +35,6 @@
 
 <header>
     <h1 class="left">
-        <img src="/photos/logo.png" id="logo">
+        <a href= "http://localhost:3000/"><img src="/photos/logo.png" id="logo"></a>
```

Comment:
@kartik1000 This is not the correct way of using the URL (localhost link). This will only work on the localhost at port 3000 and not when the port is changed or when actually deployed. Look at other `<a href= ...` how it's used, in some other files.

Reply:
@Vishal1541 ok Sir I will correct it